src/client/theme/text-styles.ts

```diff
@@ -1,100 +1,95 @@
   },
   heading2: {
-    fontFamily: 'inter',
-    fontSize: '36px',
-    fontStyle: 'normal',
-    fontWeight: 400,
-    lineHeight: '45px',
-    letterSpacing: '0px',
-  },
-  h2: {
-    fontFamily: 'inter',
-    fontSize: '24px',
-    fontStyle: 'normal',
+    fontSize: '1.5rem',
+    lineHeight: '2rem',
     fontWeight: 600,
-    lineHeight: '32px',
     letterSpacing: '-0.019em',
-    color: 'primary.500',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
-  h3: {
-    fontFamily: 'inter',
-    fontSize: '20px',
-    fontWeight: 600,
-    lineHeight: '32px',
+  heading3: {
+    fontSize: '1.125rem',
+    lineHeight: '1.5rem',
+    fontWeight: 500,
     letterSpacing: '-0.014em',
-    color: 'primary.500',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
-  sub1: {
-    fontFamily: 'inter',
-    fontWeight: 500,
-    fontSize: '16px',
-    lineHeight: '24px',
+  subhead1: {
+    fontSize: '1rem',
+    lineHeight: '1.5rem',
+    fontWeight: 600,
     letterSpacing: '-0.011em',
-    color: 'neutral.800',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
-  sub2: {
-    fontFamily: 'inter',
-    fontWeight: 400,
-    fontSize: '14px',
-    lineHeight: '24px',
+  subhead2: {
+    fontSize: '0.875rem',
+    lineHeight: '1.5rem',
+    fontWeight: 500,
     letterSpacing: '-0.006em',
-    color: 'neutral.900',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
-  sub3: {
-    fontFamily: 'inter',
+  subhead3: {
+    fontSize: '0.875rem',
+    lineHeight: '1.5rem',
     fontWeight: 600,
-    fontSize: '16px',
-    lineHeight: '24px',
     letterSpacing: '0.08em',
     textTransform: 'uppercase',
-    color: 'neutral.700',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
   body1: {
-    fontFamily: 'inter',
-    fontWeight: 'normal',
-    fontSize: '16px',
-    lineHeight: '24px',
+    fontSize: '1rem',
+    lineHeight: '1.5rem',
+    fontWeight: 400,
     letterSpacing: '-0.011em',
-    color: 'neutral.800',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
   body2: {
-    fontFamily: 'inter',
-    fontWeight: 'normal',
-    fontSize: '14px',
-    lineHeight: '20px',
+    fontSize: '0.875rem',
+    lineHeight: '1.5rem',
+    fontWeight: 400,
     letterSpacing: '-0.006em',
-    color: '#696969',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
   },
-  button2: {
-    fontSize: '14px',
+  caption1: {
+    fontSize: '0.75rem',
+    lineHeight: '1rem',
+    fontWeight: 400,
+    letterSpacing: '0em',
+    fontFeatureSettings: "'tnum' on, 'lnum' on, 'cv05' on",
+  },
+  hero: {
+    fontFamily: 'inter',
+    color: 'primary.500',
     fontWeight: 600,
-    lineHeight: '21px',
-    letterSpacing: '0.5px',
-    fontFamily: 'ssp',
+    fontSize: { base: '40px', lg: '64px' },
+    lineHeight: { base: '48px', lg: '72px' },
+    letterSpacing: '-0.022em',
+    fontFeatureSettings: "'cv05' on",
+  },
+  h1: {
```

Comment:
Can we merge this into `heading1` or rename it such that the use case is clearer?